test/unit/event_store/repository_test.rb

```diff
@@ -1,3 +1,5 @@
+require 'test_helper'
```

Comment:
This was not working locally without this.
I have no idea why it DID work in CircleCI...

Reply:
Maybe because CircleCI runs this test along others? have you tried that locally?

Reply:
Running the whole test suite? Hm, no...

Reply:
No need to run the whole suit, only run it along any other test which includes the `require`:

```
[jlledom@fedora porta:patch-update-on-rails]$ bundle exec rake test:run TEST="test/unit/event_store/repository_test.rb"
Removing :environment prerequisite from db:create
TEST='
	test/unit/event_store/repository_test.rb
'
rake aborted!
NameError: uninitialized constant EventStore
/home/jlledom/redhat/porta/test/unit/event_store/repository_test.rb:1:in `<top (required)>'
/home/jlledom/.asdf/installs/ruby/2.6.8/bin/bundle:23:in `load'
/home/jlledom/.asdf/installs/ruby/2.6.8/bin/bundle:23:in `<main>'
Tasks: TOP => test:run => test
(See full trace by running task with --trace)


[jlledom@fedora porta:patch-update-on-rails]$ bundle exec rake test:run TEST="test/unit/event_store/*.rb"
Removing :environment prerequisite from db:create
TEST='
	test/unit/event_store/*.rb
'require
# Running tests with run options --seed 45192:

...............

Finished tests in 2.652558s, 5.6549 tests/s, 9.8019 assertions/s.


15 tests, 26 assertions, 0 failures, 0 errors, 0 skips
```

Reply:
Thanks for trying this @jlledom !
Mystery solved then.

This require is still needed here IMO.